py/picca/tests/delta_extraction/data/delta_calib2.ini

```diff
@@ -31,7 +31,7 @@ num iterations = 10
 filename = $THIS_DIR/data/cat_for_clustering_plate3655.fits.gz
 
 [correction arguments 1]
-filename = $THIS_DIR/data/delta_extraction_calib/Log/delta_attributes.fits.gz
+filename = $THIS_DIR/data/delta_extraction_calib2/calib1_delta_attributes.fits.gz
```

Comment:
This change is precisely so that the second calibration does not use the results of the first calibration (right now they are the same but they might change in the future when we update the test suite)